dm_cli/import_entity.py

```diff
@@ -124,4 +129,8 @@ def import_folder_entity(
     memory_file.seek(0)
 
     package = package_tree_from_zip(data_source, memory_file, is_root=is_root, extra_dependencies=dependencies)
-    import_package_tree(package, destination, raw_package_import, resolve_local_ids, global_files)
+    try:
+        import_package_tree(package, destination, raw_package_import, resolve_local_ids, global_files)
+    except Exception as e:
```

Comment:
Is it necessary to catch this here? Should be a more general try catch further out

Reply:
import_folder_entity is used a couple of places, so this exception would have to be added to all usages in that case, right?